volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -133,8 +133,17 @@ def _skip_key_hive_entry_path(self, kcb_flags):
 
     def get_full_key_name(self) -> str:
         output = []
+        seen = set()
+
         kcb = self.KeyControlBlock
         while kcb.ParentKcb:
+            if kcb.ParentKcb.vol.offset in seen:
+                return ""
```

Comment:
Shouldn't these be `None` or something else?  I don't want the `BaseAbsentValues` sneaking in here, but it feels like we should be alerting people that things didn't work?  The downside with that is then you have to do error checking whenever you try to pull the full key name?  Just wondering of the consequences of it returning a value but blank string in case of an error?

Reply:
That function is called from `handles` on this line:

https://github.com/volatilityfoundation/volatility3/blob/develop/volatility3/framework/plugins/windows/handles.py#L313

Which, in the `else` case of that same block, the empty string is used to indicate an error:

https://github.com/volatilityfoundation/volatility3/blob/develop/volatility3/framework/plugins/windows/handles.py#L318

If it makes more sense, I can change both places (the registry extension + line 318 in handles) to send back None instead of "", then have the yield() handler do the "variable or renderers....." setup. Is that preferred? That avoids the extension API having to send back a renderer instance.

Reply:
Also, thanks for this:

"Also, if you want github to auto tag the issues these are supposed to fix, you can't just mention it in the title, you need to put "Fixes #blah" or "Closes #blah" in the bug body, please..."

I am used to GitLab where tagging the number accomplishes it.

Reply:
Yeah, I think it would be better to return None as a way of indicating an error, and then let the caller figure out what to do with it.  Also, no problem.  5;)